modules/ROOT/pages/managing-application-data-with-object-stores.adoc

```diff
@@ -19,26 +19,27 @@ Your application can also store custom integration data as well, within the limi
 [NOTE]
 ====
 * Object stores are handled in the way described in this document only for applications deployed to CloudHub workers. For applications that you deploy via the Runtime Manager to local servers, you must handle the storage of data in your own repository. See xref:deployment-strategies.adoc[Deployment Strategies] for more details.
-* All data in an object store is handled by servers in the *US-East zone*, regardless of what zone your application is deployed to. This may impact performance.
+* All data in Object Store v1 is handled by servers in the *US-East zone*, regardless of what zone your application is deployed to. This may impact performance.
+* Object Store v2 provides expanded cloud object store functionality for use with Mule 3.8.5 and later and for Mule 4.
```

Comment:
expanded how?  should we provide a link here?

Reply:
Added link and clarified. Bullet is now:
* xref:object-store::osv2-faq.adoc[Object Store v2] expands the number of cloud object store regions and object store capabilities for use with Mule 3.8.5 and later and for Mule 4.